resources/scripts/ttx_diff.py

```diff
@@ -142,17 +142,12 @@ def run_normalizer_gpos(cargo_manifest_path: Path, font_file: Path):
 
 # we had a bug where this would sometimes hang in mysterious ways, so we may
 # call it multiple times if it fails
-def try_normalizer_gpos(cargo_manifest_path: Path, font_file: Path, out_path: Path):
+def try_normalizer_gpos(normalizer_bin: Path, font_file: Path, out_path: Path):
     if not out_path.is_file():
         cmd = [
             "timeout",
```

Comment:
since you're already using subprocess.run, I suggest you get rid of this external GNU command and use use the `timeout` parameter, which should be equivalent

https://docs.python.org/3.13/library/subprocess.html#subprocess.run

Reply:
sure, I'll do that in a separate PR.